src/main/java/io/gravitee/policy/callout/CalloutHttpPolicy.java

```
@@ -134,12 +126,18 @@ private Completable executeCallOut(HttpExecutionContext ctx) {
             )
             .flatMapCompletable(calloutResponse -> processCalloutResponse(ctx, calloutResponse))
             .onErrorResumeNext(th -> {
-                if (th instanceof CalloutException && configuration.isExitOnError()) {
-                    return ctx.interruptWith(
-                        new ExecutionFailure(configuration.getErrorStatusCode()).key(CALLOUT_HTTP_ERROR).message(th.getCause().getMessage())
-                    );
+                if (!configuration.isFireAndForget()) {
+                    if (th instanceof CalloutException && configuration.isExitOnError()) {
+                        return ctx.interruptWith(
+                            new ExecutionFailure(configuration.getErrorStatusCode())
+                                .key(CALLOUT_HTTP_ERROR)
+                                .message(th.getCause().getMessage())
+                        );
+                    }
+                    return Completable.error(th);
+                } else {
+                    return Completable.complete();
```

By doing this, you are just ignoring the error in case fireAndForget is true.
but you are still waiting for an answer from the backend before continuing.

The idea of the fireAndforget is to make the call and to not block the main process

---

fixed